main.py

```diff
@@ -1,7 +1,31 @@
-import init_django_orm  # noqa: F401
-
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
-def main() -> QuerySet:
-    pass
+def main() -> QuerySet[Actor]:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to avoid the typo initially.